src/GlassCat/GlassTypes.jl

```diff
@@ -104,16 +97,21 @@ Get the name (including catalog) of the glass, or glass with this ID.
 """
 glassname(g::Glass) = glassname(g.ID)
 function glassname(ID::GlassID)
-    if ismodel(ID)
-        return "GlassCat.ModelGlass.$(num(ID))"
-    elseif isMIL(ID)
-        return "GlassCat.GlassFromMIL.$(num(ID))"
-    elseif isair(ID)
+    t, n = ID.type, ID.num
+    if t === MODEL
+        return "GlassCat.ModelGlass.$(n)"
+    elseif t === MIL
+        return "GlassCat.GlassFromMIL.$(n)"
+    elseif t === AIR
         return "GlassCat.Air"
-    elseif isother(ID)
-        return OTHER_GLASS_NAMES[num(ID)]
+    elseif t === OTHER
+        return OTHER_GLASS_NAMES[n]
+    elseif t === AGF
+        return AGF_GLASS_NAMES[n]
+    elseif t === TEST
+        return TEST_GLASS_NAMES[n]
     else
-        return AGF_GLASS_NAMES[num(ID)]
+        throw(ArgumentError("unsupported GlassID type $t"))
```

Comment:
I think this is strongly preferable to `else -> AGF`. Similarly for `glassforid()` below